src/cmd/manage_nsfs.js

```diff
@@ -61,13 +56,15 @@ async function main(argv = minimist(process.argv.slice(2))) {
             config.load_nsfs_nc_config();
             config.reload_nsfs_nc_config();
         }
-
-        global_config.accounts_dir_path = path.join(global_config.config_root, global_config.accounts_dir_name);
-        global_config.access_keys_dir_path = path.join(global_config.config_root, global_config.access_keys_dir_name);
-        global_config.buckets_dir_path = path.join(global_config.config_root, global_config.buckets_dir_name);
         global_config.config_root_backend = argv.config_root_backend ? String(argv.config_root_backend) : config.NSFS_NC_CONFIG_DIR_BACKEND;
+        global_config.fs_context = native_fs_utils.get_process_fs_context(global_config.config_root_backend);
+
+        global_config.configFS = new ConfigFS(
```

Comment:
variable naming guidelines -
```suggestion
        global_config.config_fs = new ConfigFS(
```